Author: Dominik George <nik@naturalnet.de>
Description: Do not rely on $ENV to get user's home
Forwarded: no
--- a/rssdrop
+++ b/rssdrop
@@ -6,12 +6,14 @@ use XML::Simple qw/XMLout XMLin/;
 use Getopt::Long;
 use Digest::MD5 qw/md5_hex/;
 use POSIX qw/strftime/;
+use File::HomeDir qw(home);
+use File::Spec::Functions qw(catfile);
 use LWP::UserAgent;
 use Date::Manip;
 use strict;
 
 my %opt = (
-  prefix => $ENV{RSSDROP_PREFIX} || "$ENV{HOME}/.rssdrop",
+  prefix => $ENV{RSSDROP_PREFIX} || catfile(home(), ".rssdrop"),
   fetch => 1,
 );
 
